core/common/src/main/java/alluxio/AbstractThriftClient.java

```diff
@@ -165,7 +166,8 @@
       }
     } while (retryPolicy.attemptRetry());
 
-    LOG.error("Failed after " + retryPolicy.getRetryCount() + " retries.");
+    //LOG.error("Failed after " + retryPolicy.getRetryCount() + " retries.");
+    LOG.error("Failed after {} retries.", retryPolicy.getRetryCount());
```

Comment:
We already have git for version control, so there is no need to comment out the modified code, could you please delete it?